pom.xml

```diff
@@ -739,6 +739,127 @@
             <scope>test</scope>
         </dependency>
 
+        <!-- Upper bound fix: list of transitive dependency to enforce highest version library -->
+        <dependency>
+            <groupId>com.jcraft</groupId>
+            <artifactId>jsch</artifactId>
+            <version>0.1.54</version>
+        </dependency>
+        <dependency>
+            <groupId>org.apache.httpcomponents</groupId>
+            <artifactId>httpclient</artifactId>
+            <version>4.5.3</version>
+        </dependency>
+        <dependency>
+            <groupId>org.apache.httpcomponents</groupId>
+            <artifactId>httpcore</artifactId>
+            <version>4.4.6</version>
+        </dependency>
+        <dependency>
+            <groupId>org.objenesis</groupId>
+            <artifactId>objenesis</artifactId>
+            <version>2.4</version>
+        </dependency>
+        <dependency>
+            <groupId>org.springframework</groupId>
+            <artifactId>spring-beans</artifactId>
+            <version>2.5.6.SEC03</version>
+        </dependency>
+        <dependency>
+            <groupId>org.apache.ant</groupId>
+            <artifactId>ant</artifactId>
+            <version>1.9.2</version>
+        </dependency>
+        <dependency>
+            <groupId>org.apache.maven</groupId>
+            <artifactId>maven-embedder</artifactId>
+            <version>3.5.0</version>
+        </dependency>
+        <dependency>
+            <groupId>org.eclipse.sisu</groupId>
+            <artifactId>org.eclipse.sisu.plexus</artifactId>
+            <version>0.3.3</version>
+        </dependency>
+        <dependency>
+            <groupId>org.apache.maven</groupId>
+            <artifactId>maven-core</artifactId>
+            <version>3.5.0</version>
+        </dependency>
+        <dependency>
+            <groupId>org.apache.maven</groupId>
+            <artifactId>maven-aether-provider</artifactId>
+            <version>3.3.1</version>
+        </dependency>
+        <dependency>
+            <groupId>commons-io</groupId>
+            <artifactId>commons-io</artifactId>
+            <version>2.5</version>
+        </dependency>
+        <dependency>
+            <groupId>org.jenkins-ci.plugins.icon-shim</groupId>
+            <artifactId>icon-set</artifactId>
```

Comment:
Ask @tfennelly to revisit https://github.com/jenkinsci/plugin-pom/pull/72.

Reply:
Pretty much same failure, matrix-auth has 2.0.3. It getting fixed in core won't help because blueocean is not going to upgrade to latest core. I think its easier for us to define this dependency to make things work. When we get to later version of core we can clean it up ourselves.

```
Require upper bound dependencies error for org.jenkins-ci.plugins.icon-shim:icon-set:1.0.5 paths to dependency are:
+-io.jenkins.blueocean:blueocean-rest-impl:1.3.0-beta-3-SNAPSHOT
  +-org.jenkins-ci.main:jenkins-core:2.46.3
    +-org.jenkins-ci.plugins.icon-shim:icon-set:1.0.5
and
+-io.jenkins.blueocean:blueocean-rest-impl:1.3.0-beta-3-SNAPSHOT
  +-org.jenkins-ci.plugins:matrix-auth:1.7
    +-org.jenkins-ci.plugins.icon-shim:icon-shim:2.0.3
      +-org.jenkins-ci.plugins.icon-shim:icon-set:2.0.3
```